### PR TITLE
Add info about count store

### PR DESCRIPTION
**Pull request by JPryce-Aklundh**
Trello: https://trello.com/c/ldWkUXex/5093-documentation-on-countstore

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
Looks like you've updated the documentation!

Check out your changes at https://neo4j-docs-cypher-635.surge.sh

File: modules/ROOT/pages/functions/aggregating.adoc [hunk 1]
```diff
@@ -300,6 +300,11 @@ count(expression)
 
 |===
 
+[NOTE]
+====
+Neo4j maintains a transactional count store for holding count metadata, which significantly increases the speed of queries using the `count()` function.
+More information about the count store can be read in this link:https://neo4j.com/developer/kb/fast-counts-using-the-count-store/[Neo4j Knowledge Base article].
```
Comment (stefano-ottolenghi) [inline modules/ROOT/pages/functions/aggregating.adoc]:
```suggestion
For more information about the count store, refer to link:https://neo4j.com/developer/kb/fast-counts-using-the-count-store/[Fast counts using the count store].
```

I think it's always nicer to use the page title, when it doesn't disrupt the style, so as a user I know what I'm gonna get if I click.

File: modules/ROOT/pages/functions/aggregating.adoc [hunk 2]
```diff
@@ -300,6 +300,11 @@ count(expression)
 
 |===
 
+[NOTE]
+====
+Neo4j maintains a transactional count store for holding count metadata, which significantly increases the speed of queries using the `count()` function.
```
Comment (stefano-ottolenghi) [inline modules/ROOT/pages/functions/aggregating.adoc]:
```suggestion
Neo4j maintains a transactional count store for holding count metadata, which can significantly increase the speed of queries using the `count()` function.
```

File: modules/ROOT/pages/functions/aggregating.adoc [hunk 3]
```diff
@@ -300,6 +300,11 @@ count(expression)
 
 |===
 
+[NOTE]
```
Comment (stefano-ottolenghi) [inline modules/ROOT/pages/functions/aggregating.adoc]: I think a `TIP` would be more appropriate.

**Review by stefano-ottolenghi — APPROVED** [review]

**Pull request merged**